languages/csharp/exercises/concept/numbers-floating-point/.meta/design.md

```diff
@@ -2,28 +2,45 @@
 
 ## Goal
 
-TODO
+The goal of this exercise is to teach the student how the Concept of Floating-point Numbers is implemented in [C#][docs.microsoft.com-floating-point-numeric-types]. It will show the available floating-point types and discuss their differences.
```

Comment:
Any reason for the extra capitalization here?
```suggestion
The goal of this exercise is to teach the student how the concept of floating-point numbers is implemented in [C#][docs.microsoft.com-floating-point-numeric-types]. It will show the available floating-point types and discuss their differences.
```

Reply:
The Concept one is intented (see other PR), the other one isn't. Thanks!